Gemfile

```diff
@@ -46,7 +46,7 @@ gem 'bootsnap', require: false
 # gem "sassc-rails"
 
 # Use Active Storage variants [https://guides.rubyonrails.org/active_storage_overview.html#transforming-images]
-# gem "image_processing", "~> 1.2"
+# gem 'image_processing', '~> 1.2'
 
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
```

Comment:
- [ ] Again as requested by the previous reviewer, once you have installed the expected gem, kindly make sure the expected tests for this milestone are passing.
![image](https://user-images.githubusercontent.com/81932478/214544743-628c9193-faae-4677-b6af-c36606ef170b.png)

Reply:
@Munsa1  thanks once again BUT 
I am using Request spec; when you run
 RSpec spec/requests
folder;  the tests all passed

Reply:
- [ ] Hi so even with the command you have provided I get the following error
![image](https://user-images.githubusercontent.com/81932478/214555694-2894f614-91ea-49a0-b518-74f45ca44505.png)